{"fullName": "test_weather#test_zhengzhou", "status": "broken", "start": 1582683789425, "historyId": "77d74a839c61662d63a1efe28ef82fb9", "stop": 1582683789425, "labels": [{"value": "test_weather", "name": "suite"}, {"value": "PC-20160717KIMX", "name": "host"}, {"value": "8844-MainThread", "name": "thread"}, {"value": "pytest", "name": "framework"}, {"value": "cpython3", "name": "language"}, {"value": "test_weather", "name": "package"}], "uuid": "a06f7bde-e115-41f6-a523-9c67687ed765", "testCaseId": "800ee3877b3709da617200f82b785939", "name": "test_zhengzhou", "statusDetails": {"trace": "def test_zhengzhou():\n>       assert len(wstest.wea_forecast(city_id=539)) > 10\nE       NameError: name 'wstest' is not defined\n\ntest_weather.py:15: NameError", "message": "NameError: name 'wstest' is not defined"}}